transport/errors/errormapper.go

```diff
@@ -67,6 +69,11 @@ func isConflictError(err error) bool {
 		isSvcErrorWithCode(err, svcerrors.ErrCodeReleaseDuplicateTitle)
 }
 
+func isBadRequestError(err error) bool {
+	return isSvcErrorWithCode(err, svcerrors.ErrCodeSlackChannelNotSetForProject) ||
+		isSvcErrorWithCode(err, svcerrors.ErrCodeSlackIntegrationNotEnabled)
```

Comment:
If user wants to send a release notification and slack integration is either not enabled or no slack channel is set for project, `bad request` is returned. Let me know if you would rather choose different http status code.

Reply:
Yeah, I also don't see a better option. 😕